src/main/java/org/symphonyoss/symphony/messageml/elements/Radio.java

```diff
@@ -72,23 +72,32 @@ public void buildAll(MessageMLParser context, org.w3c.dom.Element element) throw
 
   @Override
   public org.commonmark.node.Node asMarkdown() {
-    return new RadioNode();
+    return new RadioNode(getAttribute(NAME_ATTR));
```

Comment:
After reading the remaining code I understood the way you were handling markdown, however, that's not the correct way. You should instead be creating FormElementNode directly in here and passing the "name" attribute as the "text" of the node itself, instead of concatenating the constant with the name of the field later on.